### ISSUE TITLE
move Python [default] to the top of the startup menu?

### ISSUE THREAD

**Issue by hainm** [Anaconda-Platform/nb_conda_kernels#43]
Normally uses like me just want to use our default Python env.

![top](https://cloud.githubusercontent.com/assets/4451957/17338871/402db21a-58b7-11e6-854b-e8020d905ce5.png)

**Comment by damianavila** [comment]
We certainly need a better way to handle the case where people have lot of environments.
@bollwyvl thoughts on this one? I guess we can have something quick to "hide" the conda-based kernels names... or put them in something like this structure:

![menu](https://cloud.githubusercontent.com/assets/1640669/17369636/7ab5e606-596f-11e6-829e-9607f76727a1.png)

**Comment by bollwyvl** [comment]
We'll have to do that at the UI level, which we don't currently have access
to... and to change it requires monkeypatching the tree.js stuff.

I suppose we could follow what JupyterLab does, and just hack it at runtime
(gross).

I think the submenus are not a terrible idea.

**Comment by janschulz** [comment]
...or use a gross hack and add a char at the beginning which sorts it below the rest...

:-)

**Comment by damianavila** [comment]
> We'll have to do that at the UI level, which we don't currently have access
to... and to change it requires monkeypatching the tree.js stuff.

Yep, not sure to invest time on this when we have some nice ideas on #42...

**Comment by mcg1969** [comment]
Closing this for three reasons. For one it looks like the consensus is that this requires changes to the Jupyter UI, and I'd much rather Jupyter do that. For another, I made a small change in #115 that adds a trailing asterisk to the display name of the active environment, so at least it's more obvious which one is current. Finally, the short-named "Python" and "R" environments (duplicates of the default environment kernels) are easy enough to pick out of the list.
